nucypher/network/server.py

```diff
@@ -17,10 +33,14 @@
 from nucypher.keystore.keypairs import HostingKeypair
 from nucypher.keystore.threading import ThreadedSession
 from nucypher.network.protocols import InterfaceInfo
+from jinja2 import Template
 
 
+HERE = BASE_DIR = os.path.abspath(os.path.dirname(__file__))
+TEMPLATES_DIR = os.path.join(HERE, "templates")
```

Comment:
Whoops. I think, you forgot to commit the actual directory with templates?

Reply:
Great!

I think, it should also be included in `package_data` in `setup.py` (otherwise it won't be available unless with `-e`). Idk if it should be a concern of this PR or some other

Reply:
Noted - See #502